libs/@local/hash-isomorphic-utils/src/save.ts

```diff
@@ -170,8 +164,7 @@ const calculateSaveActions = async (
        *       being handled here, so that's okay for now.
        */
       if (dependedOn) {
-        const idx = actions.findIndex((item) => !item.createEntityType);
-        actions.splice(idx === -1 ? 0 : idx, 0, action);
```

Comment:
This added `dependedOn` entities to the front of the queue after `createEntityType` actions. These actions don't exist anymore, so we just 'unshift' the entity to the front now